From 5bbb42d1b61f5407bc5b352cdbb68f80b09a56b8 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Sun, 13 Nov 2022 20:45:46 +0100
Subject: [PATCH 12/14] Every voldmanaged storage is adoptable

(new version of phh patch for november 2022 release android 11)

Every voldmanaged storage is adoptable
---
 main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index 2ddb958..77c8ab5 100644
--- a/main.cpp
+++ b/main.cpp
@@ -255,7 +255,7 @@ static int process_config(VolumeManager* vm, bool* has_adoptable, bool* has_quot
             int partnum = entry.partnum;
             int flags = 0;
 
-            if (entry.is_encryptable()) {
+            if (entry.is_encryptable() || true) {
                 flags |= android::vold::Disk::Flags::kAdoptable;
                 *has_adoptable = true;
             }
-- 
2.25.1

